api/MediaDevices.json

```diff
@@ -208,40 +211,65 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/getUserMedia",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
+                "Chrome throws error if the page serving the script is loaded from insecure origin (i.e. HTTP)."
+              ]
             },
             "chrome": {
-              "version_added": null
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
+                "Chrome throws error if the page serving the script is loaded from insecure origin (i.e. HTTP)."
+              ]
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
+                "Chrome throws error if the page serving the script is loaded from insecure origin (i.e. HTTP)."
```

Comment:
Like before, I think this needs to be moved to a `"secure_context_required"` sub-feature for `getUserMedia`

Reply:
Agree